src/ui/public/agg_types/buckets/range.js

```diff
@@ -13,6 +13,24 @@ define(function (require) {
       makeLabel: function (aggConfig) {
         return aggConfig.params.field.displayName + ' ranges';
       },
+      getKey: function (bucket, key, agg) {
+        let range = { gte: bucket.from, lt: bucket.to };
+
+        if (range.gte == null) range.gte = -Infinity;
+        if (range.lt == null) range.lt = +Infinity;
+
+        return range;
+      },
+      getFormat: function (agg) {
+        if (agg.$$rangeAggTypeFormat) return agg.$$rangeAggTypeFormat;
+
+        var RangeFormat = FieldFormat.from(function (range) {
+          var format = agg.fieldOwnFormatter();
+          return `[${format(range.gte)}, ${format(range.lt)})`;
```

Comment:
This should either start with an open paren `(` or end with a close square bracket `]`. 

Right now it displays as `[2kB, 4kB)`, but `(2kB, 4kB)` or `[2kB, 4kB]` seems more correct.